src/Components/Feed/Posts.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect} from 'react'
+import { Post } from './Post.jsx'
+import axios from 'axios';
+import { WithPost } from '../../HOCs/WithPost.jsx';
+import { useDoLike } from '../apis/useDoLike.jsx';
+import { useGetPost } from '../apis/useGetPost.jsx';
+import { useDoUnLike } from '../apis/useDoUnlike.jsx';
+import { PostLoader } from '../Loaders/PostLoader.jsx';
+
+export const PostsComponent = ({ statePost, setStatePost }) => {
+  const [postRes, getPost] = useGetPost();
+  const [res, doLike] = useDoLike()
+  const [unlikeRes, doUnLike] = useDoUnLike()
+  const [currentPostId, setCurrentPostId] = useState()
+  const [errorMessage, setErrorMessage] = useState('');
+  let userId = null
```

Comment:
when is userId updated?  are we always sending null for `userId`, how's that helpful?

Reply:
resolved